touchforms/formplayer/static/formplayer/script/webformsession.js

```diff
@@ -230,7 +231,8 @@ WebFormSession.prototype.loadForm = function($form, initLang) {
         'instance-content': this.instance_xml,
         'lang': initLang,
         'session-data': this.session_data,
-        'nav': 'fao'
+        'nav': 'fao',
+        'uses-sqlite': this.uses_sqlite
```

Comment:
so this will only pass the parameter on the load of the form and not any other call, just confirming that that's fine for this